Tasks/DownloadBuildArtifact/typings/globals/node/typings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "resolution": "main",
```

Comment:
it's deprecated.  I have an action item to back through our tasks 1:1.  After I do that, I want to send as a pattern to other teams to do the same...